include/deal.II/lac/parpack_solver.h

```diff
@@ -108,10 +108,17 @@ extern "C" {
  * eigenvector/eigenvalue pairs to solve for. Here, <code>lambda</code> is a
  * vector that will contain the eigenvalues computed, <code>x</code> a vector
  * of objects of type <code>V</code> that will contain the eigenvectors
- * computed. <code>OP</code> is an inverse operation for the matrix <code>A -
+ * computed.
+ *
+ * Currently, only three modes of (P)Arpack are implemented. In mode 3,
```

Comment:
somewhere in this paragraph, I think it would be useful to say that mode 3 is the default